jsuite-frontend/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# pull official base image
+FROM node:16.15
```

Comment:
I'm not sure if this is happening on your end but I am getting the same error as this guy. When your containers are up can you access localhost:3000/ out of the box?

https://stackoverflow.com/questions/65055617/deploying-create-react-app-on-openshift-eacces-permission-denied-open-home

Reply:
This is not an issue for me. localhost:3000/ is accessible right away. Could you elaborate more on the error and what you have to do to get it to work?

Reply:
![image](https://user-images.githubusercontent.com/52092026/173959124-dc1dd84a-f932-476f-a475-742445bde10b.png)
This is what I get when trying to go straight to localhost once the containers are up

Reply:
This should be fixed with the latest commit